plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -207,6 +214,17 @@ private function get_feature_definitions() {
 					'disable_ui'         => false,
 					'is_legacy'          => true,
 				),
+				'order_source_attribution' => array(
+					'name'               => __( 'Order Source Attribution', 'woocommerce' ),
+					'description'        => __(
+						'Enable this feature to track and credit channels and campaigns that contribute to orders on your site. (Beta)',
```

Comment:
> Also, by definition of the feature being beta, should it be in the Experimental features section? Although I understand that it may be alarming to find an Experimental feature auto-enabled ¯_(ツ)_/¯

We talked about it, and decided that it even though it's beta, it doesn't fit with the warning given in Experimental features being something to "enable [...] at your own risk!"

pe2C5g-JD-p2#comment-775